<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRplusplusClass</class>
 <widget class="QMainWindow" name="QRplusplusClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QR++ : QR Code Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>QRplusplus_icon.ico</normaloff>QRplusplus_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>760</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>760</width>
      <height>360</height>
     </size>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>741</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="inputLayout">
      <item>
       <layout class="QHBoxLayout" name="textLayout">
        <item>
         <widget class="QLabel" name="label_text">
          <property name="text">
           <string>Text:</string>
          </property>
          <property name="buddy">
           <cstring>userTextInput</cstring>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pasteButton">
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="userTextInput">
        <property name="plainText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Enter your text to be put into the QR Code...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupCustomize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>600</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Customize</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="customizeLayout">
      <item row="3" column="2">
       <widget class="QComboBox" name="comboBox_color">
        <item>
         <property name="text">
          <string>Black (Default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pink</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Purple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Deep Purple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Indigo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Teal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Deep Orange</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Brown</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grey</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_errorCor">
        <property name="text">
         <string>Error Correction:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_errorlvl</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBox_errorlvl">
        <item>
         <property name="text">
          <string>Low (Default)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBox_borderSize">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_color">
        <property name="text">
         <string>Color:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_color</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_borderSize">
        <property name="text">
         <string>Border Size:</string>
        </property>
        <property name="buddy">
         <cstring>spinBox_borderSize</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>405</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Generate Button</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>140</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupDirectory">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Directory</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>741</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="directoryLayout">
      <item>
       <widget class="QLabel" name="label_saveto">
        <property name="text">
         <string>Save to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_fileDirectory"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionBrowse"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWebsite"/>
    <addaction name="actionGithub"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Nayuki"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGenerate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About QR++</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Website (Coming Soon)</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="text">
    <string>Browse...</string>
   </property>
  </action>
  <action name="actionGenerate_2">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Nayuki">
   <property name="text">
    <string>About QRcodegen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrplusplus.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>userTextInput</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pasteButton</sender>
   <signal>clicked()</signal>
   <receiver>userTextInput</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
